.pre-commit-config.yaml

```diff
@@ -26,4 +26,7 @@ repos:
     rev: 5.12.0
     hooks:
       - id: isort
-        args: ["--profile", "black", "--filter-files"]
\ No newline at end of file
+        args:
+          - --profile=black
+          - --filter-files
+          - --line-length=79
```

Comment:
Ah, 79 is the recommended in the [PEP8 standard](https://peps.python.org/pep-0008/#maximum-line-length) ! And is also the default max line length for the Flake8 linter, so the addition of this argument was to avoid conflicts between Flake8 and Black (slightly more liberal).

This config is consistent with what we've been using for the other Neurobagel Python tools, i.e. the CLI and API. See also [this discussion](https://github.com/neurobagel/bagel-cli/pull/54#discussion_r1100829265) between Seb & I on line length for the CLI 🙂